package-lock.json

```diff
@@ -0,0 +1,513 @@
+{
```

Comment:
@patrick-fs We already have a `yarn.lock` file checked in. I am OK switching to `npm` from `yarn` but if we do that, let's remove the `yarn.lock` file.

Reply:
I'll revert.

Reply:
I've removed package-lock.json